engine/flink/model-util/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/AggregatorFunction.scala

```diff
@@ -56,6 +86,7 @@ trait AggregatorFunctionMixin { self: StateHolder[TreeMap[Long, AnyRef]] =>
 
     updateState(newState, newElementInStateTimestamp + timeWindowLengthMillis, timeService)
     handleElementAddedToState(newElementInStateTimestamp, newElement, value.context, timeService, out)
+    sizeHistogram.update(newState.size)
```

Comment:
I think that we should at least add a comment that is not valid metric. Someone could think that it holds histogram of current snapshot of states for all keys, but in fact it is biased by keys that update state more frequently then other ones. Also this histogram won't come down to zero even if all states will be evictated. 
Maybe better approach would be to add extra state value `timestampOfLastHistogramUpdate` and make sure that is updated in some interval and also add `update` invocation to `evictStates`?

Reply:
I added comment and changed name, to reflect that I'm interested in how many buckets were deserialized. 
It will come to zero after 10 secs of no events ;)